README.md

```diff
@@ -2,27 +2,28 @@
 ![dbt-core](https://img.shields.io/badge/dbt_core-version_>=1.0.0_<2.0.0-orange.svg)
 
 # GitHub dbt Package ([Docs](https://dbt-github.netlify.app/))
+
 # 📣 What does this dbt package do?
-This package builds off the [dbt_github_source](https://github.com/fivetran/dbt_github_source) staging models and applies transformations that enables you to better understand your GitHub issues and pull requests.  Its main focus is to enhance these two core objects with commonly used metrics. Additionally, the metrics tables let you better understand your team's velocity over time.  These metrics are available on a daily, weekly, monthly and quarterly level. 
+This pre-built, open-source package builds on the [dbt_GitHub_source dbt package](https://github.com/fivetran/dbt_github_source). It applies transformations that enhance your GitHub issues and pull requests with common metrics so you can better understand them. Additionally, the metrics tables show your team's velocity over time. These metrics are available on a daily, weekly, monthly, and quarterly level. 
```

Comment:
You MUST run the source package before the transform package, correct? If so, we should make that clear.

Reply:
The source package will be run automatically if they are running the transform package. I don't think there is a need to make that clear since it will be performed for them.

Reply:
Got it - you're right, let's leave it as is, then.